src/app/static/src/app/store/load/load.ts

```diff
@@ -0,0 +1,25 @@
+import {actions} from "./actions";
+import {mutations} from "./mutations";
+import {Module} from "vuex";
+import {RootState} from "../../root";
+
+export enum LoadingState { NotLoading, SettingFiles, UpdatingState, LoadFailed }
```

Comment:
We might not actually need to keep track of all these states for the purposes of the front end, but it's been helpful for me to keep track of what's happening!

Reply:
might be useful for showing progress/loading spinners or something